labs/lab-4-jquery/script.js

```diff
@@ -0,0 +1,62 @@
+var party;
+var width;
+// var table;
+
+$(document).ready(function() { // not able to get this to quite work with jQuery
+   $(".form").hide();
+   var resturant = new FloorPlan(3, 3);
+   FloorPlan.render();
+   Table.render();
+});
+
+class Party {
+  constructor(partyName, partySize, partyContact) {
+    this.partyName = partyName;
+    this.partySize = partySize;
+    this.partyContact = partyContact;
+  }
+}
+
+class Table {
+  constructor(tableNum, status, party, available) {
```

Comment:
Four arguments here, but you only use two of them within this method, and you only pass *one* in on line 54.

Maybe this should be `constructor(tableNum) {`?